src/tests/functional/plugin/shared/include/behavior/plugin/core_threading.hpp

```diff
@@ -333,20 +333,6 @@ TEST_P(CoreThreadingTestsWithIterations, smoke_LoadNetworkAccuracy) {
     }, numIterations, numThreads);
 }
 
-// tested function: single IECore ReadNetwork, SetConfig, LoadNetwork, AddExtension
-TEST_P(CoreThreadingTestsWithIterations, smoke_LoadNetwork_SingleIECore) {
```

Comment:
We shouldn't remove test cases, we need to understand why it happens.

Reply:
@ilyachur @ilya-lavrenov   please see my comments in jira - 89440, this case is not thread-safe in current plugin implementation.

Reply:
It was threads safe for many releases, what was change? In it's a regression, we need to address it

Reply:
ilya, this case is newly added(https://github.com/openvinotoolkit/openvino/pull/11796/files), not the one existing for a long time.

Reply:
Is the issue seen w/o https://github.com/openvinotoolkit/openvino/pull/12240 ?

Reply:
yes, seen w/o #12240, I have found below logs in jira ticket 85384, the date is before @12440 merged. @wgzintel  have we reproduce this failures locally?
ie-tests-linux-ubuntu20-cpu/59462/consoleText-[2022-07-05T01:28:21.642Z] [2022-07-05 01:28:21,469] [139666179897088]
CPU/CoreThreadingTestsWithIterations.smoke_LoadNetwork_SingleIECore/targetDevice=CPU_config=PERF_COUNT:YES_numThreads=4_numIter=50
Killed
--
ie-tests-linux-ubuntu20-cpu/59458/consoleText-[2022-07-05T00:02:15.199Z] [2022-07-05 00:02:14,948] [140252173358848]
CPU/CoreThreadingTestsWithIterations.smoke_LoadNetwork_SingleIECore/targetDevice=CPU_config=PERF_COUNT:YES_numThreads=4_numIter=50
Killed

Reply:
This failures did not reproduce locally, I have tried to run 500 times, no killed.

Reply:
discussed offline, the test is invalid, you are right.